src/components/Discussion/Internal/Comment/CommentEmbed.js

```diff
@@ -85,20 +86,23 @@ const dateFormat = timeFormat('%d.%m.%Y %H:%M')
 
 const normalizeEmbed = embed => ({
   ...embed,
-  imageUrl: embed.imageUrl || embed.image,
-  header: embed.siteName || embed.userName,
-  headerImageUrl: embed.siteImageUrl || embed.userProfileImageUrl,
+  imageUrl: embed?.imageUrl ?? embed?.image,
+  header: embed?.siteName ?? embed?.userName,
+  headerImageUrl: embed?.siteImageUrl ?? embed?.userProfileImageUrl,
   html:
-    embed.__typename === 'TwitterEmbed' &&
-    embed.html &&
-    embed.html.replace(/\s*target="_blank"/g, ''),
-  body: embed.description
+    embed?.__typename === 'TwitterEmbed' &&
+    embed?.html &&
+    embed?.html.replace(/\s*target="_blank"/g, ''),
+  body: embed?.description
 })
 
-export const Embed = ({ comment }) => {
+const propTypes = {
+  embed: PropTypes.object.isRequired,
+  mentioningDocument: PropTypes.object
+}
+
+export const CommentEmbed = ({ embed, mentioningDocument }) => {
   const [colorScheme] = useColorContext()
-  if (!comment || !comment.embed) return null
```

Comment:
this file now has crazy amounts of defensive coding, but lake simple checks like this. let's fix this the next time